src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.css

```diff
@@ -0,0 +1,35 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  margin-top: 15px;
+  margin-bottom: 100px;
+}
+
+.title {
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+}
+
+.description {
+  margin-bottom: 18px;
+  font-size: var(--size-tiny);
+  line-height: 18px;
+  color: color-mod(var(--temp-grey-blue-7) alpha(85%));
```

Comment:
Weight for description is bold

Spec
![Screenshot from 2021-04-20 13-55-40](https://user-images.githubusercontent.com/1193222/115387601-98e93300-a1e3-11eb-9892-fb0785489032.png)

Yours
![Screenshot from 2021-04-20 13-55-42](https://user-images.githubusercontent.com/1193222/115387603-9981c980-a1e3-11eb-9700-1d63165e74cd.png)